force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -94,17 +115,31 @@ public with sharing class DummyJSONCallout {
 	
 	public static void postCreateDummyJSONUser(String contactId) {
 		// Create HTTP request to send.
-
+        HttpRequest request = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+        request.setEndpoint('callout:DummyJsonUser' + 'add');
 		// Set the HTTP method to POST.
-
+        request.setMethod('POST');
 		// Set the body using generateDummyJsonUserPayload method.
-
+        request.setBody(generateDummyJsonUserPayload(contactId));
 		// Send the HTTP request and get the response.
-
+       Http http = new Http();
+	   HttpResponse response = http.send(request);
 		// If the HTTP response code is successful, update the contact.
+		if (response.getStatusCode() >= 200 || response.getStatusCode() <= 299) {
+			//Map<String, Object> responseBody = (Map<String, Object>) JSON.deserializeUntyped(response.getBody());
+			//Integer dummyId = (Integer) responseBody.get('id');
+			Contact cont = [SELECT Id, DummyJSON_Last_Updated__c FROM Contact WHERE Id = :contactId LIMIT 1];
```

Comment:
If you have the contact Id, you dont need to query first to get the record, you can just construct it with the id. `Contact cont = new Contact(Id = contactId);`